tests/lwip/Makefile

```diff
@@ -1,10 +1,25 @@
 include ../Makefile.tests_common
 
+LWIP_IPV4 ?= 0
+
+ifneq (0, $(LWIP_IPV4))
+  USEMODULE += ipv4_addr
+  USEMODULE += lwip_arp
+  USEMODULE += lwip_ipv4
+  USEMODULE += lwip_dhcp
```

Comment:
```suggestion
  USEMODULE += lwip_dhcp_auto
```

:wink:

Reply:
I had it already in my local version but I wanted to wait until #12965 is merged to avoid compile errors.